new_arrivals_chi/app/database.py

```diff
@@ -0,0 +1,146 @@
+    password = db.Column(db.String(100), nullable=False)
+    organization_id = db.Column(db.Integer, db.ForeignKey("organizations.id"))
+    organization = db.relationship("Organization", back_populates="users")
+
+
+class Organization(db.Model):
+    __tablename__ = "organizations"
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(255), nullable=False)
+    location_id = db.Column(
+        db.Integer, db.ForeignKey("locations.id"), nullable=False
+    )
+    hours_id = db.Column(db.Integer, db.ForeignKey("hours.id"), nullable=False)
+    phone = db.Column(db.String(25), nullable=False)
+    image_path = db.Column(db.String(255), nullable=True)
+    status = db.Column(db.String(50), nullable=False)
+
+    # Relationships
+    users = db.relationship("User", back_populates="organizations")
+    languages = db.relationship(
+        "Language",
+        back_populates="organizations",
+    )
+    services = db.relationship(
+        "Service",
+        back_populates="organizations",
+    )
+    hours = db.relationship("Hours", back_populates="organizations")
+
+
+class Language(db.Model):
+    __tablename__ = "languages"
+    id = db.Column(db.Integer, primary_key=True)
+    language = db.Column(db.String(50), nullable=False)
+    organizations = db.relationship(
+        "Organization",
+        back_populates="language",
+    )
+
+
+class Hours(db.Model):
+    __tablename__ = "hours"
+    id = db.Column(db.Integer, primary_key=True)
+    day_of_week = db.Column(db.Integer, nullable=False)
+    opening_time = db.Column(db.Time, nullable=False)
+    closing_time = db.Column(db.Time, nullable=False)
+    organizations = db.relationship("Organization", back_populates="hours")
+
+
+class Service(db.Model):
+    __tablename__ = "services"
+    id = db.Column(db.Integer, primary_key=True)
+    category = db.Column(db.String(100), nullable=False)
+    service = db.Column(db.String(100), nullable=False)
+    access = db.Column(db.String(100), nullable=False)
+    service_note = db.Column(db.String(255), nullable=True)
+
+    organizations = db.relationship(
+        "Organization",
+        back_populates="services",
+    )
+
+    service_dates = db.relationship(
+        "ServiceDate",
+        back_populates="services",
+    )
+
+
+class ServiceDate(db.Model):
+    __tablename__ = "service_dates"
+    id = db.Column(db.Integer, primary_key=True)
+    date = db.Column(db.Date, nullable=False)
+    start_time = db.Column(db.Time, nullable=False)
+    end_time = db.Column(db.Time, nullable=False)
+    repeat = db.Column(
+        Enum(
+            "every day",
+            "every week",
+            "every month",
+            "every other week",
+            name="repeat_types",
+        ),
+        nullable=False,
+    )
+    service_id = db.Column(db.Integer, db.ForeignKey("services.id"))
+    service = db.relationship(
+        "Service",
+        back_populates="service_dates",
+    )
```

Comment:
also should there be a reference to location ? lmk your thoughts!

Reply:
If all `ServiceDate`s take place at a specific location, I think it's worth adding. You could potentially make it nullable in case the respective location isn't present yet for the service date.